### PR TITLE
SLCORE-7 All issues are reported with severity MAJOR

### PR DESCRIPTION
**Pull request by henryju**

### CODE REVIEW

File: src/main/java/org/sonarsource/sonarlint/core/container/rule/RuleFinderCompatibility.java [hunk 1]
```diff
@@ -111,12 +112,14 @@ private static Rule toRule(@Nullable RulesDefinition.Rule ar) {
...
-  private static Rule toRuleNotNull(RulesDefinition.Rule ar) {
-    Rule rule = Rule.create(ar.repository().key(), ar.key())
-      .setName(ar.name())
-      .setLanguage(ar.repository().language())
-      .setConfigKey(ar.internalKey());
-    for (Param param : ar.params()) {
+  private static Rule toRuleNotNull(RulesDefinition.Rule ruleDef) {
+    Rule rule = Rule.create(ruleDef.repository().key(), ruleDef.key())
+      .setName(ruleDef.name())
+      .setSeverity(RulePriority.valueOf(ruleDef.severity()))
```
Comment (SonarTech) [inline src/main/java/org/sonarsource/sonarlint/core/container/rule/RuleFinderCompatibility.java]: MINOR Remove this use of "RulePriority"; it is deprecated. [rule](https://dory.sonarsource.com/coding_rules#rule_key=squid%3ACallToDeprecatedMethod)

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 issue:
* MINOR 1 minor

Watch the comments in this conversation to review them.

**Comment by SonarTech** [comment]
SonarQube analysis reported 1 issue:
* MINOR 1 minor

Watch the comments in this conversation to review them.

**Comment by dbmeneses** [comment]
LGTM

**Pull request merged**